dataactcore/models/baseInterface.py

```diff
@@ -17,7 +17,7 @@ class BaseInterface(object):
     Session = None
     engine = None
     session = None
-
+    interfaces = None
```

Comment:
Minor, but it might be helpful to reference where this is being set (i.e. `InterfaceHolder`) in a comment to make the dependency a bit more explicit